enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/constants/RouterAccessLogConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.enforcer.constants;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * RouterAccessLogConstants class holds the constants used in Router access logs
+ */
+public class RouterAccessLogConstants {
+    public static final String ORIGINAL_PATH_DATA_NAME = "originalPath";
+
+    public static final List<String> QUERY_PARAMS_TO_REMOVE_FROM_PATH = Arrays.asList("apikey");
```

Comment:
APIKey query param can be customized. Hence adding it as a constant will not work in our case.